src/pages/Admin/Venue/Rooms/RoomsForm.tsx

```diff
@@ -320,29 +322,20 @@ const RoomInnerForm: React.FC<RoomInnerFormProps> = (props) => {
                   </div>
                   <div className="toggle-room">
                     <div className="input-title">Enabled ?</div>
-                    <label className="switch">
-                      <input
-                        disabled={disable}
-                        type="checkbox"
-                        id="isEnabled"
-                        name={"isEnabled"}
-                        ref={register}
-                      />
-                      <span className="slider round"></span>
-                    </label>
+                    <Toggler
+                      id="isEnabled"
+                      name="isEnabled"
+                      forwardedRef={register}
+                      disabled={disable}
+                    />
                   </div>
                   <div className="toggle-room">
                     <div className="input-title">Is label hidden?</div>
```

Comment:
```suggestion
                    {/* @debt pass the header into Toggler's 'label' prop instead of being external like this*/}
                    <div className="input-title">Is label hidden?</div>
```